src/control/server/storage/scm/ipmctl_region.go

```diff
@@ -128,22 +140,42 @@ func (rh regionHealth) MarshalXML(e *xml.Encoder, start xml.StartElement) error
 	return e.EncodeElement(ipmctl.PMemRegionHealth(rh).String(), start)
 }
 
-// constants for ipmctl commandline calls
-//
-// Manage AppDirect/Interleaved memory allocation goals across all DCPMMs on a system.
 const (
-	cmdShowIpmctlVersion = "ipmctl version"
-	cmdShowRegions       = "ipmctl show -o nvmxml -region" // returns region info in xml
-	cmdCreateRegions     = "ipmctl create -f -goal%sPersistentMemoryType=AppDirect"
-	cmdRemoveRegions     = "ipmctl create -f -goal%sMemoryMode=100"
-	cmdDeleteGoals       = "ipmctl delete -goal"
+	ipmctlName = "ipmctl"
 
 	outNoCLIPerms    = "ipmctl command you have attempted to execute requires root privileges"
 	outNoPMemModules = "No PMem modules in the system"
 	outNoPMemRegions = "no Regions defined in the system"
+	// Command output when specifying socket in cmd and no regions for that socket.
+	outNoPMemRegionResults = "<Results>\n<Result>\n</Result>\n</Results>"
```

Comment:
I'm cautious of relying on the xml to be formatted a certain way (e.g. with newlines and no whitespace/tabs/etc.). IMO it would be better to process the xml and just find that nothing is there. That said, the ipmctl tool will probably not be changing much in the future, so this is not urgent to change.

Reply:
This was to handle a strange edge case, if there are no regions and `ipmctl show -o nvxml -region` is called then "no Regions" appears on stdout whereas if `ipmctl show -o nvxml -region -socket 1` is called you get the results xml as above. I wll leave it as is for the moment if that's okay.

Reply:
I'm fine with leaving as-is for now, since it seems unlikely for ipmctl to change anything in the near future.